cloud/azure/modules/app/locals.tf

```diff
@@ -3,11 +3,61 @@ locals {
   # because the private_dns_zone_configs and record_sets blocks expose lists, even if we only have one dns zone
   # and record set configured.
   postgres_private_link = azurerm_private_endpoint.endpoint.private_dns_zone_configs[0].record_sets[0].fqdn
-  generated_hostname    = "${var.application_name}-${random_pet.server.id}.azurewebsites.net"
 
   postgres_password_keyvault_id = "postgres-password"
   app_secret_key_keyvault_id    = "app-secret-key"
   adfs_secret_keyvault_id       = "adfs-secret"
   aws_secret_access_token       = "aws-secret-access-token"
   aws_access_key_id             = "aws-access-key-id"
+
+  app_settings = {
```

Comment:
this stayed the same minus moving to locals correct?

Reply:
also why locals over just living in the main?

Reply:
I had to remove `BASE_URL` but other than that, it's the same.

Reply:
this block was originally in main bc it was used by the app service resource, but now that it's being used by the main deployment and the canary slot I moved it to locals